codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -141,12 +138,15 @@ class ServerProtocolTestGenerator(
                 // These tests are broken because they are missing a target header.
                 FailingTest.RequestTest(AWS_JSON_10, "AwsJson10ServerPopulatesNestedDefaultsWhenMissingInRequestBody"),
                 FailingTest.RequestTest(AWS_JSON_10, "AwsJson10ServerPopulatesDefaultsWhenMissingInRequestBody"),
+                FailingTest.RequestTest(REST_JSON, "RestJsonServerPopulatesDefaultsWhenMissingInRequestBody"),
                 // Response defaults are not set when builders are not used https://github.com/smithy-lang/smithy-rs/issues/3339
                 FailingTest.ResponseTest(AWS_JSON_10, "AwsJson10ServerPopulatesDefaultsInResponseWhenMissingInParams"),
                 FailingTest.ResponseTest(
                     AWS_JSON_10,
                     "AwsJson10ServerPopulatesNestedDefaultValuesWhenMissingInInResponseParams",
                 ),
+                FailingTest.ResponseTest(REST_JSON, "RestJsonServerPopulatesDefaultsInResponseWhenMissingInParams"),
+                FailingTest.ResponseTest(REST_JSON, "RestJsonServerPopulatesNestedDefaultValuesWhenMissingInInResponseParams"),
```

Comment:
`RestJsonServerPopulatesDefaultsInResponseWhenMissingInParams` and `RestJsonServerPopulatesNestedDefaultValuesWhenMissingInInResponseParams` fail because of https://github.com/smithy-lang/smithy-rs/issues/3339. The protocol test generator does not use the builder to instantiate the struct, but instead uses direct struct instantiation with an expression, hence the `@default` values don't get filled out.

Reply:
Can you add a TODO above these two linking to #3339?

Reply:
Actually, a TODO with 3339 is already at line 141. I'll update that TODO to explicitly state that the TODO applies to the following 4 tests.